mlir/test/Integration/Dialect/XeGPU/SIMT/simple_gemm.mlir

```diff
@@ -0,0 +1,121 @@
+// RUN: mlir-opt %s --gpu-lower-to-xevm-pipeline="xegpu-op-level=workitem" \
```

Comment:
Please change the directory and pipeline parameter to "lane", instead of SIMT and workitem, to be consistent with XeGPU layout definition.

Reply:
Thanks a lot, Jianhui. Changed it now.